extensions/tmtv/src/Panels/PanelROIThresholdSegmentation/ROIThresholdConfiguration.tsx

```diff
@@ -60,18 +60,24 @@ function ROIThresholdConfiguration({ config, dispatch, runCommand }) {
       </div>
 
       {config.strategy === ROI_STAT && (
-        <Input
+        <InputNumber
+          id="percentage-max-suv"
           label={t('Percentage of Max SUV')}
           labelClassName="text-[13px] font-inter text-white"
-          className="border-primary-main bg-black"
-          type="text"
-          containerClassName="mr-2"
+          className="mr-2"
+          inputContainerClassName="border-primary-main bg-black rounded-[4px]"
+          inputClassName="text-white bg-black text-[14px]"
           value={config.weight}
-          onChange={e => {
+          minValue={0}
+          maxValue={1}
+          step={0.01}
```

Comment:
We did not have adjustment arrows before so we don't need them now. Please hide the adjustment arrows.